client/src/js/agreement/AgreementForm.jsx

```diff
@@ -61,13 +79,13 @@ function AgreementForm (props) {
     if (props.agreementType === AgreementType.INDIVIDUAL) {
       agreement = new Agreement(
         props.agreementType === AgreementType.INDIVIDUAL ? AgreementType.INDIVIDUAL : AgreementType.CORPORATE,
-        'TODO, add agreement body',
+        ClaText,
         signer
       )
     } else if (props.agreementType === AgreementType.CORPORATE) {
       agreement = new Agreement(
         props.agreementType === AgreementType.INDIVIDUAL ? AgreementType.INDIVIDUAL : AgreementType.CORPORATE,
-        'TODO, add agreement body',
+        ClaText,
```

Comment:
Is the corporate text expected to be the same as the individual one? If not, we should open a GitHub issue to add both.

Reply:
I thought the multiple CLA was not to be addressed for this release and tracked here: https://github.com/OpenNetworkingFoundation/cla-manager/issues/41

But we can probably have two different ones for Individual and Corporate